pkg/cmd/create/create_pullrequest.go

```diff
@@ -86,6 +87,7 @@ func NewCmdCreatePullRequest(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().StringArrayVarP(&options.Labels, "label", "l", []string{}, "The labels to add to the pullrequest")
 	cmd.Flags().BoolVarP(&options.Push, "push", "", false, "If true the contents of the source directory will be committed, pushed, and used to create the pull request")
 	cmd.Flags().BoolVarP(&options.Fork, "fork", "", false, "If true, and the username configured to push the repo is different from the org name a PR is being created against, assume that this is a fork")
+	cmd.Flags().BoolVarP(&options.UpdateExisting, "update-existing", "", false, "If true, lookup existing PRs based on the labels passed in and update an existing PR if one is found")
```

Comment:
How is this related to "force pushing" a tag? +1 for being able to update an existing PR, but how is this related to force pushing.

Reply:
I'll create another issue to capture the pull request not updating